buildlib/azure-pipelines.yml

```diff
@@ -15,6 +15,33 @@ resources:
       endpoint: ucfconsort_registry
 
 stages:
+  - stage: LatestCCs
+    jobs:
+      - job: latest_cc
+        displayName: Latest CCs and CppCheck
+        container: fedora
+        steps:
+          - bash: ./autogen.sh
+            displayName: Setup autotools
+
+          - bash: ./contrib/configure-release
+            displayName: Configure
+
+          # this will also generate compile_commands.json for cppcheck
+          - bash: bear make -j`nproc` V=1
+            displayName: GCC
+
+          - bash: |
+              make clean
+              export CC=clang; export CXX=clang++
+              ./contrib/configure-release
```

Comment:
minor question: why not just
```
./contrib/configure-release CC=clang; export CXX=clang++
```
it will remember compiler values in makefiles & will use it in build

Reply:
Just used the approach I got used to. If you see any significant benefits of your option (except shortness), let me know and I'll update.

Reply:
one of the benefits (not strong, but at least something :) ) - it is clean environment space after task is completed, no exported variables which may affect next tasks